### PR TITLE
PhpUnitMethodCasingFixer - Add option to use non breaking spaces

### PR DESCRIPTION
**Pull request by julienfalque**

### CODE REVIEW

**Comment by kubawerlos** [comment]
[![wat]](https://mnapoli.fr/using-non-breakable-spaces-in-test-method-names/)

[wat]: https://mnapoli.fr/images/posts/nbsp-wat.jpg

**Comment by julienfalque** [comment]
Is that a :+1:? :)

File: tests/Fixer/PhpUnit/PhpUnitMethodCasingFixerTest.php [hunk 1]
```diff
@@ -36,29 +36,122 @@ public function testFixToCamelCase($expected, $input = null)
...
+     */
+    public function testFixFromNonBreakingSpacesToCamelCase($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
 
+    /**
+     * @dataProvider provideCamelCaseNonBreakingSpacesFixCases
+     *
+     * @param mixed      $expected
```
[the remark this answers is not in the document]
Comment (julienfalque) [reply tests/Fixer/PhpUnit/PhpUnitMethodCasingFixerTest.php]: Fixed.

[the remark this answers is not in the document]
Comment (kubawerlos) [reply tests/Fixer/PhpUnit/PhpUnitMethodCasingFixerTest.php]: @julienfalque what about `@param null|mixed $input`? `$input` is nullable string, right?

[the remark this answers is not in the document]
Comment (julienfalque) [reply tests/Fixer/PhpUnit/PhpUnitMethodCasingFixerTest.php]: Indeed, fixed as well.

**Comment by SpacePossum** [comment]
Im sorry, but not a fan of this notation... is it commonly used?

**Comment by julienfalque** [comment]
I would say not much, but I don't actually know.

**Comment by kubawerlos** [comment]
For the record, I've approved it because as @keradus wrote:

> code is perfect

My feelings are described with the picture from my 1st comment - BTW you can click it ;)

**Comment by julienfalque** [comment]
Closing because of the lack of interest from the community. Using `testdox` output, PHPUnit replaces underscores with spaces anyway so I'm not even sure I'll continue using this syntax myself.

**Pull request closed without merging**